### ISSUE TITLE
12$: Report nice error for destructuring assignment without parentheses

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#470]
The following code looks valid, but it is invalid:
```js
let point = {x: 3, y: 4};
let x, y;
{x, y} = point;
console.log(x, y);
```
quick-lint-js correctly reports an error, but it is unhelpful:
> hello.js:4:8: error: missing operand for operator [E026]

It should instead report that `(` `)` is missing around the assignment:
> hello.js:4:1: error: missing parentheses around destructuring assignment
